sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -44,6 +44,10 @@
 #include <level_zero/zes_api.h>
 #include <sycl/detail/iostream_proxy.hpp>
 
```

Comment:
Could you please avoid adding executable mode to this file? Github says the mode is being changed from `100644` to `100755`